common/source/docs/common-camera-shutter-with-servo.rst

```diff
@@ -22,67 +21,51 @@ This article explains what settings you need to configure for both servos and re
+   You can monitor and log *exactly* when the was
    camera triggered. For more information see the 
    :ref:`Enhanced camera trigger logging <common-camera-shutter-with-servo_enhanced_camera_trigger_logging>` 
    section below.
 
-First set the camera trigger output type using the ``CAM_TRIGG_TYPE``
-setting:
-
--  Open *Mission Planner* and then click on **CONFIG/TUNING \| Full
-   Parameters List**.
--  Set :ref:`CAM_TRIGG_TYPE` to 0 for a servo (output PWM signal) or 1 for a 
-   :ref:`relay <common-relay>` (Note: although ArduPilot supports
-   multiple relay channels only the first relay can be used as a camera
-   trigger). The image below shows the camera trigger set as a servo.
-
-   .. figure:: ../../../images/cam_trigg_dist1.jpg
-      :target: ../_images/cam_trigg_dist1.jpg
-
-      Mission Planner: Configuring Camera Trigger Output Type
-
-The actual port used for the shutter is set and configured in the
-*Camera Gimbal Configuration Screen*:
-
--  Open **Initial setup \| Optional Hardware \| Camera Gimbal**. The
-   shutter settings are shown in the section at the bottom.
+The actual output used for the shutter is set and configured in the SETUP/Optional Hardware/Camera Gimbal screen:
 
    .. figure:: ../../../images/missionplannercameragimbalscreen.jpg
       :target: ../_images/missionplannercameragimbalscreen.jpg
 
       Mission Planner: Camera Gimbal Configuration Screen
 
--  The *Shutter* drop-down list is used to set the connected camera
-   trigger port. Here we have selected RC10, which corresponds to AUX2
+-  The **Shutter** drop-down list is used to set the connected output for camera
+   trigger. Here we have selected SERVO10, which corresponds to AUX2
    on the Pixhawk.
--  The Shutter *Duration* setting specifies how long the servo/relay
-   will be held in the *Pushed* state when the shutter is activated, in
+-  The **Duration** setting specifies how long the servo/relay
+   will be held in the **Pushed** state when the shutter is activated, in
    tenths of a second. Above the value is 10, so the pushed state is
-   held for one second.
+   held for one second. **Not Pushed** when the shutter is not active.
 -  **For Servos only (settings ignored for relay outputs):**
 
-   -  The Shutter *Pushed* and *Not Pushed* settings hold PWM signal
+   -  The Shutter *Pushed* and *Not Pushed* settings are the PWM signal
       values that will be sent when the servo is in those states.
    -  The *Servo Limits* setting specifies the range of PWM signal
       values within which the servo will not bind.
-
       
+.. note:: Mission Planners screen is not up-to-date in that the request to "Please set the Ch7 Option to Camera Trigger" is out of date, and the "transistor" selection in the *Shutter* drop-down list does nothing.
+
+- To set which RC Channel will act as the manual shutter release, configure its ``RCx_OPTION`` in the CONFIG/Full Parameter List or CONFIG/User Params to "Camera Trigger".
```

Comment:
I think "act as" should be replaced with "control" to make it more clear that the RC input channel controls what happens.